physics-notes/chapters/2D-3D-Motion/2d-3d-motion.tex

```diff
@@ -0,0 +1,156 @@
+\documentclass{scrartcl}
+
+\usepackage{amsthm}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{siunitx}
+
+\begin{document}
+    \section{Motion in Two and Three Dimensions}
+    \paragraph{Position Vector} It is a vector that extends from a reference point to a particle. In unit vector notation:
+    \begin{equation}
+        \vec{r} = x\hat{i}+y\hat{j}+z\hat{k}
+    \end{equation}
+    \paragraph{Displacement} As the particle moves, the position vector changes. The particle's displacement \vec{\Delta{r}} is:
+    \begin{equation}
+        \Delta\vec{r}= \vec{r_2} - \vec{r_1}
+    \end{equation}
+    Or, in unit vector notation:
+    \begin{equation}
+        \vec{\Delta{r}} = (x_2 - x_1) \hat{i} + (y_2 - y_1) \hat{j} + (z_2 - z_1) \hat{k}
+    \end{equation}
+    \paragraph{Average Velocity} When a particle moves through a displacement-r}}$ in a time interval ${\Delta{t}}$ , its average velocity is: 
```

Comment:
Error still stands, what were you trying to write here?

Reply:
I beleive he meant it without "}}$"  in line 22

Reply:
All the enhance i suggest replacing with a simple bold ... i'll take care of it later in the day

Reply:
Mmm, looking closer at the rest of the text i'd say it was a ${\Delta{r}}$ (which can normally be a $\Delta r$, dunno why he does it with so many {}), anyway I can do that rn man- it will take 5 mins

Reply:
(also because looking at it it needs a little bit of factorization, look at the \subparagraph{\small{...}} things, they won't comply with the rest of the production)
(also replacing all the ${\Delta{...}}$ with some $\Delta ...$)